### ISSUE TITLE
[User flows and navigating] Link or button is not working as expected or did not navigate to the intended destination (03.11.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28332]
#### VFS team name
Healthcare Experience

#### VFS product name
Check-in MVP

#### Point of Contact/Reviewers
Mikki Northuis

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
The "< Back to last screen" link does not work on any of the pages it is displayed on.  

### Supporting Guidance
[Category 0, Issue 0](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### VFS Team Tasks to Complete

- [x] Comment on the ticket if there are questions or concerns
- [x] VFS team closes the ticket when the issue has been resolved

**Comment by jparker0425** [comment]
This has been corrected and in staging.

**Comment by shiragoodman** [comment]
updating labels so ticket is consistent with post-pilot process. Also updated title to reflect correct VA.gov Experience Standard 03.11

**Comment by jparker0425** [comment]
This item was taken care of and is functioning in production.
